client/components/cart.js

```diff
@@ -1,96 +1,85 @@
-import React from 'react'
+import React, {useEffect} from 'react'
 import {
   fetchItems,
-  addAnItem,
+  addAnItem, // do we need this as part of the cart?
```

Comment:
I would say probably not. Even if we implement a "Similar Items" sidebar, we would want the user to go to the single product view for that item before adding it to their cart.